src/gatsby-node.js

```diff
@@ -13,8 +12,6 @@ import {
 } from './utils';
 
 const ComponentNodeTypes = new Set(['PloneBreadcrumbs', 'PloneNavigation']);
-const DownloadableContentTypes = new Set(['Image', 'File']);
-
 // TODO: Make DownloadableContentTypes configurable
```

Comment:
Probably also this comment should be removed.

Meanwhile I should go to git blame to see why I did add that...

Reply:
ok

Reply:
@iFlameing That comment is still relevant, but it should be moved to utils.js next to the DownloadableContentTypes definition.

Reply:
ok! I will make this change.